addons/killtracker/XEH_postInit.sqf

```diff
@@ -136,9 +136,18 @@ GVAR(killCount) = 0;
                 _unitName = format ["*AI* - %1", getText ((configOf _unit) >> "displayName")];
             };
         };
-        if (_unitIsPlayer || GVAR(trackAI)) then {
-            TRACE_3("send kill event",_killer,_unitName,_killInfo);
-            [QGVAR(kill), [_unitName, _killInfo], _killer] call CBA_fnc_targetEvent;
+        TRACE_3("send kill event",_instigator,_unitName,_killInfo);
+        [QGVAR(kill), [_unitName, _killInfo], _instigator] call CBA_fnc_targetEvent;
+
+        if (GVAR(showCrewKills) && {!(_killer isKindof "CAManBase")}) then {
+            private _crew = [driver _killer, gunner _killer, commander _killer] - [_instigator];
```

Comment:
I haven't tested, but I presume that the crew will be informed if a passenger in an FFV  seat gets a kill. Is that wanted behaviour/do we care? Do we want to ensure that the instigator is in the driver, gunner or commander seat?

Reply:
ffv kills won't trigger this,
only kills made by vehicle's weapons